gitclone/git.go

```diff
@@ -279,21 +279,37 @@ func checkout(gitCmd git.Git, arg, branch string, depth int, isTag bool) error {
 		}
 		return gitCmd.Fetch(opts...)
 	}); err != nil {
-		return fmt.Errorf("Fetch failed, error: %v", err)
+		return &StepError{
```

Comment:
Note: using [NewStepError](https://github.com/bitrise-steplib/steps-activate-ssh-key/pull/18/files#diff-de54b22b0d36aeb746ac186f364820b4R10) constructor seemed to be more feature proof (enforces more to set required properties).
Might worth moving into a shared package.

Reply:
Replaced with NewStepError.